hanadb_exporter/exporter_factory.py

```diff
@@ -19,19 +20,37 @@ class SapHanaExporter(object):
 
     Args:
         exporter_type (str): Exporter type. Options: prometheus
+        metrics_file (str): Path to the metrics file
+        hdb_connector (hdb_connector.HdbConnector): SAP HANA database connector
     """
 
+    @staticmethod
+    def get_hana_version(connector):
+        """
+        Query the SAP HANA database version
+
+        Args:
+            connector: HANA database api connector
+        """
+        query = 'SELECT * FROM sys.m_database;'
+        query_result = connector.query(query)
```

Comment:
In case the DB is down for X reason, what does `connector.query` return?

Do you have a upstream doc for the API I mean the python db, i coulnd' find it.. :grin:

Reply:
@MalloZup Here the docs. The query will fail, and the exporter won't start. It looks like a proper action in my opinion (we could improve the output log maybe)

Reply:
ok thx!